include/libopencm3/ethernet/mac_stm32fxx7.h

```diff
@@ -58,8 +58,10 @@
 #define ETH_MACVLANTR			MMIO32(ETHERNET_BASE + 0x1C)
 #define ETH_MACRWUFFR			MMIO32(ETHERNET_BASE + 0x28)
 #define ETH_MACPMTCSR			MMIO32(ETHERNET_BASE + 0x2C)
-/* not available on F1 ?*/
-#define ETH_MACDBGR			MMIO32(ETHERNET_BASE + 0x34)
+/* Following registers not available on F1 */
+#if !defined(STM32F1)
```

Comment:
we haven't normally done much work trying to prevent this sort of thing unless htere's been conflicts, as far as the _definitions_ go.  at least, the comment becomes pretty irrelevant :)

Reply:
Should I just change all of these exceptions to notes rather than checking if STM32F1 is defined?

As far as I can tell, there are no register conflicts between the F1 and F4/7 peripheral registers. In fact, I think it's possible that the registers may work on the F1 and they are just not documented. In every other respect the hardware is identical, but I only have F4/F7 here to test.

This PR is only fixing one register definition that is actually wrong, the rest is just flagging where certain registers and register locations are not documented in the STM32F1 reference manual. As I have no way of testing whether these undocumented registers work or not on F1 targets, is it worth me just restricting this PR to the one incorrect register and removing the `#if !defined(STM32F1)` parts?

Reply:
Sounds good to me.